src/Nordea/Components/InformationDetails.elm

```diff
@@ -60,6 +82,66 @@ card attrs children title =
             ]
 
 
+collapsibleCard :
+    { attrs : List (Attribute msg)
+    , title : String
+    , emphasisedText : String
+    , isOpen : Bool
+    , children : List (Html msg)
+    }
+    -> Html msg
+collapsibleCard { attrs, title, emphasisedText, isOpen, children } =
```

Comment:
Should we instead have only one card and have it regular/collapsible based on `isCollapsible` parameter? This would be a small change but not sure if it's convenient since the collapsible card needs more/different parameters than regular one.

Reply:
I think it sounds like a good idea to merge the two. Regarding the parameters, what about taking an optional record?

```elm
card :
    { attrs : List (Attribute msg)
    , title : String
    , collapsibleProps: Maybe { emphasisedText : Html msg, isOpen: Bool }
    , children : List (Html msg)
    }
```

Reply:
I'm thinking how to do merge safely. Should we do the merge but leave old and new one and start slowly using the merged one in PH? Do you know if InformationDetails regular card is used outside of PartnerHub?

Reply:
I would say do the merge in one go. We don't have to start using the collapsible feature all over at first, but we can set it to Nothing and get the old behaviour. I think they may also be used in the customer portal, maybe esign. But it shouldn't be a problem, they can set the prop to Nothing as well when they update.

Reply:
Yeah okay. I'll do that 👍